.github/workflows/doc.yml

```
@@ -53,6 +53,27 @@ jobs:
     - name: 'Build HTML documentation'
       run: make -C Doc/ SPHINXOPTS="-q" SPHINXERRORHANDLING="-W --keep-going" html
 
+    # Add pull request annotations for Sphinx nitpicks (missing references)
+    - name: Get list of changed files
+      id: changed_files
+      uses: Ana06/get-changed-files@v2.2.0
+    - name: 'Build changed files with nitpicks'
```

Let's use the exact spelling from `sphinx-build --help`:
```
  -n                nit-picky mode, warn about all missing references
```
```suggestion
    - name: 'Build changed files in nit-picky mode'
```